src/common/utils.tsx

```diff
@@ -188,17 +199,22 @@ export const clarityValuetoHumanReadable = (value: any) => {
     return value.repr;
   }
   return null;
-  // const deserializeAsset = deserializeCV(Buffer.from(value.replace('0x', ''), 'hex'));
-  //
-  // if (deserializeAsset.type === 5 && 'address' in deserializeAsset) {
-  //   return addressToString(deserializeAsset.address);
-  // }
-  // if (deserializeAsset.type === 1 && 'value' in deserializeAsset) {
-  //   return (deserializeAsset.value as BN).toString();
-  // }
-  // if (deserializeAsset.type === 2 && 'buffer' in deserializeAsset) {
-  //   return (deserializeAsset.buffer as Buffer).toString();
-  // }
+};
+
+export const addSepBetweenStrings = (strings: (string | undefined)[], sep: string = '∙') => {
```

Comment:
Can you comment this function with an example?

Reply:
```tsx
const caption = addSepBetweenStrings(["Confirmed 5 days ago", "0xadsadadsa", "You viewed 1 min ago"], "/")

// caption = Confirmed 5 days ago / 0xadsadadsa / You viewed 1 min ago
```